### PR TITLE
Add Ravenor Davion leaderboard result

### PR DESCRIPTION
**Pull request by rdavion**
Final validation loss: 3.21173

  Learning curve: https://wandb.ai/rdavion-stanford/cs336-owt/reports/CS336-A1-Leaderboard-Submission-rdavion--VmlldzoxNjU4NDQzMg

  Description:
  weight tying, Muon, bf16 autocast, torch.compile, tuned hyperparameters

### CODE REVIEW

**Comment by marcelroed** [comment]
Oops, seems like the link is broken. Can you make it public and test that it works in an incognito window?

**Comment by rdavion** [comment]
Just fixed!

**Comment by marcelroed** [comment]
Hmm, still broken on my end: 
image
